src/Console/ConfigurationResolver.php

```diff
@@ -305,11 +305,15 @@ public function getDirectory()
     {
         if (null === $this->directory) {
             $path = $this->getCacheFile();
-            $filesystem = new Filesystem();
+            if (null === $path) {
+                $absolutePath = $this->cwd;
```

Comment:
can you elaborate on this change?
it does not look related to `Event` at first glance

Reply:
@keradus `Filesystem::isAbsolutePath()` does not accept `null` anymore, so that case is handled separately.

Reply:
ah. thanks for clarification !